src/analysis/utils/runtime-utils.ts

```diff
@@ -128,15 +128,16 @@ export const getConvertedRuntimeStatus = (runtime: Runtime | undefined): LeoRunt
 };
 
 export const getDisplayRuntimeStatus = (status: LeoRuntimeStatus): DisplayRuntimeStatus =>
-  Utils.switchCase(
+  Utils.switchCase<string, DisplayRuntimeStatus>(
     _.lowerCase(status),
     ['leoreconfiguring', () => 'Updating'],
     ['starting', () => 'Resuming'],
     ['stopping', () => 'Pausing'],
     ['stopped', () => 'Paused'],
     ['prestarting', () => 'Resuming'],
     ['prestopping', () => 'Pausing'],
-    [Utils.DEFAULT, () => _.capitalize(status)]
+    // TODO: Is this cast valid?
```

Comment:
yeah, 'running' applies here as well

Reply:
Updated this comment in https://github.com/DataBiosphere/terra-ui/pull/4154/commits/56adec16706736fd19bc90f3997035c3b9d5c61c.